system/assets/js/admin/lib/plugins/jquery.fontawesome-iconpicker.js

```diff
@@ -0,0 +1,995 @@
+                    p = b.left - c.collisionPosition.marginLeft + l + m + n - h;
+                    if (p > 0 || e(p) < k) {
+                        b.left += l + m + n;
+                    }
+                }
+                a.ui.pos._trigger(b, c, "posCollided", "flipLeft");
+            },
+            top: function(b, c) {
+                a.ui.pos._trigger(b, c, "posCollide", "flipTop");
+                var d = c.within, f = d.offset.top + d.scrollTop, g = d.height, h = d.isWindow ? d.scrollTop : d.offset.top, i = b.top - c.collisionPosition.marginTop, j = i - h, k = i + c.collisionHeight - g - h, l = c.my[1] === "top", m = l ? -c.elemHeight : c.my[1] === "bottom" ? c.elemHeight : 0, n = c.at[1] === "top" ? c.targetHeight : c.at[1] === "bottom" ? -c.targetHeight : 0, o = -2 * c.offset[1], p, q;
+                if (j < 0) {
+                    q = b.top + m + n + o + c.collisionHeight - g - f;
+                    if (b.top + m + n + o > j && (q < 0 || q < e(j))) {
+                        b.top += m + n + o;
+                    }
+                } else if (k > 0) {
+                    p = b.top - c.collisionPosition.marginTop + m + n + o - h;
+                    if (b.top + m + n + o > k && (p > 0 || e(p) < k)) {
+                        b.top += m + n + o;
+                    }
+                }
+                a.ui.pos._trigger(b, c, "posCollided", "flipTop");
+            }
+        },
+        flipfit: {
+            left: function() {
+                a.ui.pos.flip.left.apply(this, arguments);
+                a.ui.pos.fit.left.apply(this, arguments);
+            },
+            top: function() {
+                a.ui.pos.flip.top.apply(this, arguments);
+                a.ui.pos.fit.top.apply(this, arguments);
+            }
+        }
+    };
+    (function() {
+        var b, c, d, e, f, g = document.getElementsByTagName("body")[0], h = document.createElement("div");
+        b = document.createElement(g ? "div" : "body");
+        d = {
+            visibility: "hidden",
+            width: 0,
+            height: 0,
+            border: 0,
+            margin: 0,
+            background: "none"
+        };
+        if (g) {
+            a.extend(d, {
+                position: "absolute",
+                left: "-1000px",
+                top: "-1000px"
+            });
+        }
+        for (f in d) {
+            b.style[f] = d[f];
+        }
+        b.appendChild(h);
+        c = g || document.documentElement;
+        c.insertBefore(b, c.firstChild);
+        h.style.cssText = "position: absolute; left: 10.7432222px;";
+        e = a(h).offset().left;
+        a.support.offsetFractions = e > 10 && e < 11;
+        b.innerHTML = "";
+        c.removeChild(b);
+    })();
+})(presideJQuery);
+
+(function(a) {
+    "use strict";
+    if (typeof define === "function" && define.amd) {
+        define([ "jquery" ], a);
+    } else if (window.jQuery && !window.jQuery.fn.iconpicker) {
```

Comment:
I think all references to `window.jQuery` should be replaced with `window.presideJQuery` here